setup.py

```diff
@@ -121,8 +121,8 @@ def get_cmake_path():
     raise Exception("CMake must be installed to build from source.")
 
 
-use_openssl = os.environ.get('AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO', False)
-
+def using_system_libcrypto():
+    return 'AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO' in os.environ
```

Comment:
This would be `True` for `AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO=0` though. I think the alternative is better:

```python
return os.getenv('AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO') == '1'
```

It is also already used for example here: https://github.com/awslabs/aws-crt-python/blob/238d38627d925e16a403d184f2b1aef495de9889/awscrt/_test.py#L109